marimbabot_behavior/src/marimbabot_behavior/behavior_node.py

```diff
@@ -40,6 +40,14 @@ def __init__(self):
         # action client to send the note_sequence to the lilypond_audio action server
         self.lilypond_audio_client = actionlib.SimpleActionClient('audio_from_lilypond', LilypondAudioAction)
 
+        # Waits until the action server has started up and started
+        # while not self.planning_client.wait_for_server(timeout=rospy.Duration(2)) and not rospy.is_shutdown():
+        #     rospy.loginfo('Waiting for the planning action server to come up')
+
+        # Waits until the action server has started up and started
+        while not self.lilypond_audio_client.wait_for_server(timeout=rospy.Duration(2)) and not rospy.is_shutdown():
```

Comment:
I prefer to setup a timeout is better, just increase the sec to be long enough. but it's fine.

Reply:
Not sure I understand. Which timeout are you referring to? All this timeout (2s) does, is defining a time interval, in which the message "Waiting for the ... server to come up" is printed over and over again until the server is reached. It just serves as an indication to us if there are issues reaching the servers.
The actual time it takes for the servers to start is not in any way restricted by it. Currently, if a server does not ever start, this message would be printed every 2s until the code is shutdown.